src/careamics/losses/lvae/losses.py

```diff
@@ -168,30 +168,30 @@ def get_kl_divergence_loss(
         dim=1,
     )  # shape: (B, n_layers)
 
+    # Apply free bits (& batch average)
+    kl = free_bits_kl(kl, free_bits_coeff)  # shape: (n_layers,)
```

Comment:
There seem to be two definitions of this function in the current code base:

- https://github.com/federico-carrara/careamics/blob/b29fc6c95c65026c6169fbb35f1f9b4f31383568/src/careamics/losses/lvae/loss_utils.py#L4
- https://github.com/federico-carrara/careamics/blob/b29fc6c95c65026c6169fbb35f1f9b4f31383568/src/careamics/models/lvae/utils.py#L407

Could you take this opportunity to clean that up?

Reply:
Thanks for noticing.  I will clear it up :)